sailfish-types/src/message.rs

```diff
@@ -220,14 +220,18 @@ pub struct Payload<T: Committable> {
     round: RoundNumber,
     source: PublicKey,
     data: T,
+    /// when sailfish delivers to timeboost, round evidence prevents malicious timeboost nodes
+    /// placing messages arbitrarily far into the future, causing unbounded buffer grow.
```

Comment:
```suggestion
```
The sailfish crates are independent of timeboost. *If* a doc comment is absolutely necessary here it should be phrased in a way that describes the field in the context of `Payload`.

Reply:
got it, removed.